getting-started/CONCEPTS/STREAMING.md

```diff
@@ -208,6 +210,74 @@ planner=ActionPlanner(
 
 ```
 
+## Custom `StreamingResponse` Management
+
+### Definitions
+
+- **Stream ID**: A unique identifier for the stream, assigned after the initial update is sent. It ensures that all subsequent messages are associated with the correct stream.
+- **Sequence Numbers**: Each message in the stream is assigned a sequence number (`streamSequence`), starting from 1 and incrementing with each message. This helps maintain the order of messages.
+
+
+The class also maintains an internal queue for outgoing activities to ensure they are sent in order and without overloading the Teams client. 
+
+### Implementation
+
+#### 1. Initialize `StreamingResponse`
+
+To start using the `StreamingResponse` class, you need to create an instance of it within your bot's turn handler. 
+
+   ```javascript
+   const streamingResponse = new StreamingResponse(context);
+   ```
+
+
+#### 2. Send Informative Updates
+
+   ```javascript
+   streamingResponse.queueInformativeUpdate('Searching for the information you requested...');
+   ```
+
+#### 3. Queuing Text Chunks
+
+As your bot generates or retrieves content, you can send partial messages to the user.
+
+**Note**: It is important to include a delay between the chunks to avoid spamming the Teams Client.  
+
```

Comment:
I think it would be good around here to comment that we don't manage chunking for the user, and that typically comes from whatever streaming server is being used.

Then a further note that the below chunks will appear to arrive simultaneously because of lack of delay and how big the chunks are.